CHANGELOG.md

```diff
@@ -1,16 +1,32 @@
 ## Unreleased
 
+FEATURES:
+* Support Kubernetes health probe synchronization with Consul for connect injected pods via `connectInject.healthChecks` [[GH-651](https://github.com/hashicorp/consul-helm/pull/651)].
+  The default behavior for this feature is `enabled: true`.
+  See [https://www.consul.io/docs/k8s/health](https://www.consul.io/docs/k8s/health) for more information.
```

Comment:
This link doesn't resolve yet, but I'm assuming that's just coming in the future?

Reply:
Good catch, there is an open PR here for it : https://github.com/hashicorp/consul/pull/8819 
I can hold off on merging this until it gets merged.